x-pack/plugins/security_solution/public/detections/components/alerts_kpis/common/components.tsx

```diff
@@ -25,24 +25,41 @@ export const KpiPanel = styled(EuiPanel)<{ height?: number }>`
   }
 `;
 interface StackedBySelectProps {
-  selected: AlertsStackByField;
-  onSelect: (selected: AlertsStackByField) => void;
+  selected: string;
+  onSelect: (selected: string) => void;
 }
 
-export const StackBySelect: React.FC<StackedBySelectProps> = ({ selected, onSelect }) => {
-  const setSelectedOptionCallback = useCallback(
-    (event: React.ChangeEvent<HTMLSelectElement>) => {
-      onSelect(event.target.value as AlertsStackByField);
+export const StackByComboBoxWrapper = styled.div`
+  min-width: 350px;
+`;
+
+export const StackByComboBox: React.FC<StackedBySelectProps> = ({ selected, onSelect }) => {
+  const onChange = useCallback(
+    (options) => {
```

Comment:
is `options === selectedOption`?

Reply:
no, it's defined by eui here https://github.com/elastic/eui/blob/main/src/components/combo_box/combo_box.tsx#L102